data/sql/updates/pending_db_world/repandspawn.sql

```diff
@@ -0,0 +1,4 @@
+--lots of Reputation changes
+UPDATE `creature_onkill_reputation` SET `RewOnKillRepFaction1`=87, `RewOnKillRepFaction2`=21, `MaxStanding1`=5, `IsTeamAward1`=0, `RewOnKillRepValue1`=5, `MaxStanding2`=7, `IsTeamAward2`=1, `RewOnKillRepValue2`=-25, `TeamDependent`=0 WHERE `creature_id`=26081;
```

Comment:
```suggestion
DELETE FROM `creature_onkill_reputation` WHERE `creature_id`=26081;
INSERT INTO `creature_onkill_reputation` (`creature_id`, `RewOnKillRepFaction1`, `RewOnKillRepFaction2`, `MaxStanding1`, `IsTeamAward1`, `RewOnKillRepValue1`, `MaxStanding2`, `IsTeamAward2`, `RewOnKillRepValue2`, `TeamDependent`) VALUES (26081, 87, 21, 5, 0, 5, 7, 1, -25, 0);
```

Reply:
just fyi. 26081 entry does not exsist currently in the db,  for that creature_onkill_reputation table.
updating something that does not exsist will lead to a error.
the above is my proposed delete and insert query.

Reply:
Thanks for the corrections